test/test_um2netcdf.py

```diff
@@ -740,3 +749,54 @@ def test_fix_level_coord_skipped_if_no_levels(z_sea_rho_data, z_sea_theta_data):
     m_cube = mock.Mock(iris.cube.Cube)
     m_cube.coord.side_effect = iris.exceptions.CoordinateNotFoundError
     um2nc.fix_level_coord(m_cube, z_sea_rho_data, z_sea_theta_data)
+
+
+# tests - fix pressure level data
+
+def test_fix_pressure_levels_no_pressure_coord(get_fake_cube_coords):
+    cube = get_fake_cube_coords()
+
+    with pytest.raises(iris.exceptions.CoordinateNotFoundError):
+        cube.coord("pressure")  # ensure missing 'pressure' coord
+
+    um2nc.fix_pressure_levels(cube)  # should just exit
```

Comment:
Should the `None` return be asserted here?

Reply:
Yes it should, since the fix pressure levels calling code is relying on a `cube` or a value to resolve as `False`.

Good catch!

Reply:
Fixed in dd9c18dd.